frontend/src/components/mine/Compliance/MineComplianceInfo.js

```diff
@@ -0,0 +1,138 @@
+import moment from "moment";
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { Row, Col } from "antd";
+import { bindActionCreators } from "redux";
+import { connect } from "react-redux";
+
+import Loading from "@/components/common/Loading";
+import NullScreen from "@/components/common/NullScreen";
+import { getMineComplianceInfo } from "@/selectors/complianceSelectors";
+import { fetchMineComplianceInfo } from "@/actionCreators/complianceActionCreator";
+/**
+ * @class  MineTenureInfo - all tenure information related to the mine.
+ */
+
+const propTypes = {
+  mine: PropTypes.object.isRequired,
+  updateMineRecord: PropTypes.func.isRequired,
+  fetchMineRecordById: PropTypes.func.isRequired,
+  fetchMineComplianceInfo: PropTypes.func.isRequired,
+  mineComplianceInfo: PropTypes.object,
+  match: PropTypes.object.isRequired,
+};
+
+const defaultProps = {
+  mine: {},
+};
+
+class MineComplianceInfo extends Component {
+  state = { isLoading: true };
+
+  componentDidMount() {
+    this.props
+      .fetchMineComplianceInfo(this.props.mine.mine_detail[0].mine_no)
+      .then(() => this.setState({ isLoading: !this.state.isLoading }));
+  }
+
+  render() {
+    // TODO need to work out how and what is being returned.
+    return (
+      <div>
+        {this.state.isLoading && (
+          <div>
+            <Loading />
+          </div>
```

Comment:
I don't think the added div is needed. 

```
{this.state.isLoading && <Loading />}
```
This should work!

Reply:
changed and works.